examples/nat_scapy_tests.py

```diff
@@ -0,0 +1,231 @@
+#!/usr/bin/python
+# coding: latin-1
+
```

Comment:
This is very cool, and using a packet generator adds useful flexibility over using the OS's network stack, as you pointer out.
Could you add a header that gives a high-level view of the script's function and structure?
Also could add the usual copyright+license info, as I had added here:
https://github.com/niksu/pax/blob/master/examples/nat_topo.py#L6